src/app.ts

```diff
@@ -38,27 +40,34 @@ const main = async () => {
   });
 
   // Setup Firebase Admin
-  if (process.env.NODE_ENV === 'production') {
-    admin.initializeApp({
-      credential: admin.credential.cert(process.env.FCM_AUTH_KEY_PATH),
-    });
-  }
+  admin.initializeApp({
```

Comment:
why is this no longer covered by a production environment variable?

Reply:
I wasn't sure why it should be in the first place, as it makes testing notifications in dev not possible

Reply:
Readded